src/transformers/models/llama/tokenization_llama.py

```diff
@@ -24,7 +24,9 @@
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
 
 import sentencepiece as spm
+from sentencepiece import SentencePieceProcessor
```

Comment:
Out of interest - why import directly here instead of using `spm.SentencePieceProcessor` as before?

Reply:
No particular reason, will revert 😉